engines/aws/app/models/aws/quota.rb

```diff
@@ -1,7 +1,10 @@
 module AWS
-  class Quota
-    def cli
-      @cli ||= AWS::Cli.load()
+  class Quota < Quota
```

Comment:
should it be renamed to `Quota < AbstractQuota` or something different than `Quota < Quota` ?

Reply:
What's wrong with `Quota < Quota` ? Obviously it doesn't inherit from itself, so there _must_ be an upstream class to inherit from...